lib/createRef.spec.lua

```diff
@@ -0,0 +1,12 @@
+return function()
+	local createRef = require(script.Parent.createRef)
+
+	it("should create refs", function()
+		expect(createRef()).to.be.ok()
+	end)
+
+	it("should support tostring on refs", function()
+		local ref = createRef()
+		expect(tostring(ref)).to.equal("RoactReference(\"nil\")")
+	end)
```

Comment:
If we're going to test `tostring` on these, let's also verify that they take on the name of their `current` value if set, too!

Reply:
That's fair, done!